conf/formsStack.conf

```diff
@@ -25,8 +25,9 @@ config.formsKnot {
 }
 
 ########### Adapter configurations ###########
-config.formsAdapter {
-  options.config {
-    address = ${global.address.forms.example.adapter}
-  }
-}
+# example adapter config
+# config.formsAdapter {
+#   options.config {
+#     address = ${global.address.forms.example.adapter}
```

Comment:
```suggestion
 #     address = ${global.address.forms.test.adapter}
```

Reply:
Why test?
This is example, how adapter config could look like.
Maybe it should not be there, since this adapter doesn't exists and this lines are only to show example config?